docs/data/data-grid/filtering/filtering.md

```diff
@@ -309,14 +309,70 @@ The example below demonstrates how to achieve server-side filtering.
 
 ## Quick filter
 
-The grid does not natively include quick filtering.
-However, it can be implemented as in the demo below.
+The package also provides a component `<GridToolbarQuickFilter/>` which filters rows without specifying the concerned columns.
+By default, the quick filter considers the input as a list of values separated by space and keeps only rows that contain all the values.
 
 {{"demo": "QuickFilteringGrid.js", "bg": "inline", "defaultCodeOpen": false}}
 
-> ⚠️ This feature isn't natively implemented in the grid package. It's coming.
->
-> 👍 Upvote [issue #2842](https://github.com/mui/mui-x/issues/2842) if you want to see it land faster.
+### Custom filtering logic
+
+The logic quick filter logic can be switched to filter rows that contain _at least_ one value instead for each of them.
```

Comment:
```suggestion
The logic used for quick filter can be switched to filter rows that contain _at least_ one of the values specified, instead of testing if it contains all of them.
```